src/pytest_dbfixtures/factories/mysql.py

```diff
@@ -104,7 +105,7 @@ def mysql_proc_fixture(request):
         mysql_host = host or config.mysql.host
         mysql_params = params or config.mysql.params
 
-        tmpdir = path(gettempdir())
+        tmpdir = path(mkdtemp(prefix="pytest-mysql-"))
```

Comment:
`mkdtemp` creates a directory inside the one returned by `gettempdir` (unless the `dir` parameter is used to override it).